react/Carousel.tsx

```diff
@@ -287,21 +283,24 @@ export default class Carousel extends Component<Props, State> {
         <Slider
           classes={{
             root: styles.sliderRoot,
-            sliderFrame: styles.sliderFrame
+            sliderFrame: styles.sliderFrame,
           }}
           arrowRender={showArrows && this.ArrowRender}
           currentSlide={currentSlide}
           onChangeSlide={this.handleChangeSlide}
           arrowsContainerComponent={showArrows && this.ArrowContainerRender}
           duration={500}
         >
-          {banners.map(
-            (banner, i) => (
-              <Slide className={styles.slide} key={i} style={{ maxHeight: height }} sliderTransitionDuration={500}>
-                <Banner height={height} {...banner} />
-              </Slide>
-            )
-          )}
+          {banners.map((banner, i) => (
+            <Slide
+              className={styles.slide}
+              key={i}
+              style={{ maxHeight: height }}
+              sliderTransitionDuration={500}
```

Comment:
Shouldn't it be a defaultProp? 🤔

Reply:
This is not actually a prop, but it could be a new thing in the schema I think.

Reply:
I think we should keep this variable in an global variable

Reply:
Got it